### DISCUSSION TITLE
Sonarr - v4

### DISCUSSION THREAD

**Discussion by Fineline19** [tteck/Proxmox#1476] [Q&A] [answered]
Hi

Is it possible to update to the develop (v4) branch of sonarr?

Thanks for your help and these scripts.
Fineline

**Comment by MrSaiclops** [comment]
I would also like to know the answer to this? It seems that you cannot just change your branch when going to V4. "v3 non-docker installs cannot be upgraded directly to v4 and require installing Sonarr v4"

**Comment by tteck** [comment, marked as answer]
https://github.com/tteck/Proxmox/discussions/1738
